docs/UserGuide.md

```diff
@@ -100,30 +100,26 @@ Examples:
 
 Deletes the specified student from TutorBuddy
 
-Format: `delete_student STUDENT_NAME` <br>
-* Deletes the student with the specified name
-* `STUDENT_NAME` must have an exact match to the name in the student profile
+Format: `delete_student INDEX` <br>
 
 Examples:
-* `list_student` followed by `delete_student 2` deletes the 2nd student in the address book.
-* `find_student Betsy` followed by `delete_student 1` deletes the 1st student in the results of the find_student command.
+* `delete_student 2` deletes the 2nd student in the address book
 
 ### Listing all tuition sessions: `list_session`
 
 Shows a list of all tuition sessions in TutorBuddy
 
 Format: `list_session`
 
-### Locating tuition session by student name / date: `find_session`
+<!-- COMMENT OUT FOR FIND SESSION -->
+<!--### Locating tuition session by student name and session index: `find_session`
 
 Find tuition sessions that match the keyword given
 
-Format: `find_session KEYWORD`
+Format: `find_session s/STUDENT_NAME i/SESSION_INDEX`
```

Comment:
Ohya actually is find_session part of v1.2? Cos now that I think about it, its quite weird for find_session to include the Index. Like it wouldn't make sense for me to look through the whole list of session and try to get the individual index and then run find_session again, cos the information we give from the session list and find_session would be exactly the same. It would make more sense for it to be just find_session s/STUDENT_NAME. But if this is not under v1.2, I think we should comment this out first before we discuss again?

Reply:
Yup, I agree. something we can keep in consideration beyond v1.2.

Reply:
Oh and as for the comment out, the chunk (from line 114 to 138) is commented out using HTML <!-- --> tags! So should be irrelevant for now.

Reply:
Ah ok! Didn't see that the comment header start at line 114.